src/test/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommandTest.java

```diff
@@ -21,24 +20,20 @@ class CreateCollectionCommandTest {
 
   @Inject Validator validator;
 
-  @Nested
-  class Validation {
```

Comment:
Warning for Profile on Outer class, not inner: remove single-use `@Nested`.
Apologies for huge diff resulting.